src/redturtle/prenotazioni/scripts/notify_upcomming_bookings.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+"""Send reminders script"""
+
+from logging import getLogger
+
+from AccessControl.SecurityManagement import newSecurityManager
+from plone import api
+from Testing.makerequest import makerequest
+from transaction import commit
+from zope.component.hooks import setHooks
+from zope.component.hooks import setSite
+from zope.globalrequest import getRequest
+
+logger = getLogger(__name__)
+
+
+def main(app):
+    app = makerequest(app)
+    portal = app.Plone
+    admin = app.acl_users.getUserById("admin")
+
+    newSecurityManager(None, admin)
+    setHooks()
+    setSite(portal)
+
+    with api.env.adopt_user(username="admin"):
```

Comment:
are we sure that admin is in every website? With "adopt_role" doesn't work?

Reply:
tell me site where this is no true

Reply:
Solved